source/gui/settingsDialogs.py

```diff
@@ -5961,6 +5962,242 @@ def onSave(self):
 			self._providerSettings.onSave()
 
 
+class MagnifierPanel(SettingsPanel):
+	# Translators: This is the label for the magnifier settings panel.
+	title = _("Magnifier")
+	helpId = "MagnifierSettings"
+
+	def makeSettings(self, settingsSizer):
+		sHelper = guiHelper.BoxSizerHelper(self, sizer=settingsSizer)
+
+		# ZOOM SETTINGS
+		# Translators: The label for a setting in magnifier settings to select the default zoom level.
+		defaultZoomLabelText = _("Default &zoom level:")
+
+		# Create zoom level choices from 1.0 to 10.0 with 0.5 steps
+		zoomValues = [i / 10.0 for i in range(10, 101, 5)]  # 1.0 top 10.0 steps of 0.5
+		zoomChoices = [f"{value:.1f}x" for value in zoomValues]
+
+		self.defaultZoomList = sHelper.addLabeledControl(defaultZoomLabelText, wx.Choice, choices=zoomChoices)
+		self.bindHelpEvent("magnifierDefaultZoom", self.defaultZoomList)
+
+		# Store zoom values for later use
+		self.zoomValues = zoomValues
+
+		# FILTER SETTINGS
+		# Translators: The label for a setting in magnifier settings to select the default filter
+		defaultFilterLabelText = _("Default &filter:")
+		try:
+			from magnifier.utils.filterHandler import filter
+		except ImportError:
+			filter = None
+		filterChoices = [f.name.lower() for f in filter]
+		self.defaultFilterList = sHelper.addLabeledControl(
+			defaultFilterLabelText, wx.Choice, choices=filterChoices
+		)
+		self.bindHelpEvent("magnifierDefaultFilter", self.defaultFilterList)
+
+		# FULLSCREEN MODE SETTINGS
+		# Translators: The label for a setting in magnifier settings to select the default fullscreen mode
+		defaultFullscreenModeLabelText = _("Default &fullscreen mode:")
+		try:
+			from magnifier.fullscreenMagnifier import FullScreenMode
+		except ImportError:
+			FullScreenMode = None
+		fullscreenModeChoices = [mode.name.lower() for mode in FullScreenMode] if FullScreenMode else []
```

Comment:
This needs to be translated

```suggestion
		fullscreenModeChoices = [mode.displayString for mode in FullScreenMode] if FullScreenMode else []
```